### PR TITLE
Fix missing walletAddress and update README

### PR DESCRIPTION
**Pull request by Danziger**

### CODE REVIEW

File: components/WeaveTransfer/index.tsx [hunk 1]
```diff
@@ -8,6 +8,8 @@ import Button from '../Button'
 import styled from 'styled-components';
 import { Othent } from 'othent'
 
+// TODO: Rewrite using KMS?
```
Comment (Danziger) [inline components/WeaveTransfer/index.tsx:11]: Yes.

File: components/WeaveTransfer/index.tsx [hunk 2]
```diff
@@ -77,33 +61,34 @@ const WeaveTransfer = () => {
...
+
       const formData = new FormData();
       formData.append("transaction_id", transaction.transactionId);
       formData.append("sendToEmail", sendToEmail);
       formData.append("sendFromEmail", user_details.email);
-    
+
       fetch('https://server.othent.io/weavetransfer', {
         method: 'POST',
         body: formData,
```
Comment (7i7o) [inline components/WeaveTransfer/index.tsx:94]: We should update this code to use the new version of the SDK

**Review by 7i7o — APPROVED** [review]
tACK 👍 

WeaveTransfer is the only section that still needs updating

File: components/WeaveTransfer/index.tsx (hunk 2 above)
Comment (Danziger) [reply components/WeaveTransfer/index.tsx:94]: I created a [new issue](https://github.com/Othent/landing-new/issues/7) for this as I don't want to put much time into this site as it's likely going away soon. Once there's some clarity around that, I'll see what to do with WeaveTransfer (e.g. move it to the playground repo).

**Pull request merged**